api/terraform/update_api_docs/iam.tf

```diff
@@ -0,0 +1,4 @@
+module "ecs_update_api_docs_iam" {
+  source    = "github.com/wellcomecollection/terraform-aws-ecs-service.git//task_definition/modules/iam_role?ref=v1.5.0"
```

Comment:
Bikeshed - why do we have more than just the singe modules in this repo. It feels like we're using something that is _piece_ of that module, rather than just implementing the module, creating a potential coupling situation like before?

Reply:
Not a bikeshed! The answer is that actually we shouldn't and I should remove this; I ended up consolidating it into the `single_container` module and this bit is redundant.

Reply:
Agree with @alexwlchan, this is a good point not a bikeshed @jamesgorrie.